glusterd2/volgen2/volgen.go

```diff
@@ -29,6 +29,10 @@ type Entry struct {
 	// option remote-subvolume gfproxyd-<volname>
 	ExtraData  map[string]string
 	SubEntries []Entry
+	// IgnoreOptions represents list of options which should not be added in the
+	// generated Volfile. For example, bitd and scrubd both uses same xlator, if
+	// scrubber = on, then it becomes scrub daemon else it becomes bitd.
+	IgnoreOptions map[string]int
```

Comment:
Suggestion: As values are not consumed, this map can be `map[string]bool` or be a set (using map with [emptry struct](https://dave.cheney.net/2014/03/25/the-empty-struct) as value) like `map[string]struct{}`

Reply:
makes sense. I will update the code